sdk/containerregistry/azure-containerregistry/tests/test_container_registry_client.py

```diff
@@ -860,27 +874,35 @@ def test_set_audience(self, containerregistry_endpoint):
     @acr_preparer()
     @recorded_by_proxy
     def test_list_tags_in_empty_repo(self, containerregistry_endpoint):
+        repo = self.get_resource_name("repo")
+        self.import_image(containerregistry_endpoint, ALPINE, [repo])
         with self.create_registry_client(containerregistry_endpoint) as client:
-            # cleanup tags in ALPINE repo
-            for tag in client.list_tag_properties(ALPINE):
-                client.delete_tag(ALPINE, tag.name)
+            # cleanup tags in repo
+            for tag in client.list_tag_properties(repo):
+                client.delete_tag(repo, tag.name)
             
-            response = client.list_tag_properties(ALPINE)
+            response = client.list_tag_properties(repo)
             if response is not None:
                 for tag in response:
                     pass
+            
+            client.delete_repository(repo)
     
     @acr_preparer()
     @recorded_by_proxy
     def test_list_manifests_in_empty_repo(self, containerregistry_endpoint):
+        repo = self.get_resource_name("repo")
+        self.import_image(containerregistry_endpoint, ALPINE, [repo])
         with self.create_registry_client(containerregistry_endpoint) as client:
-            # cleanup manifests in ALPINE repo
-            for tag in client.list_tag_properties(ALPINE):
-                client.delete_manifest(ALPINE, tag.name)
-            response = client.list_manifest_properties(ALPINE)
+            # cleanup manifests in repo
+            for tag in client.list_tag_properties(repo):
+                client.delete_manifest(repo, tag.name)
+            response = client.list_manifest_properties(repo)
             if response is not None:
                 for manifest in response:
                     pass
+            
+            client.delete_repository(repo)
```

Comment:
I believe there is no need to delete - it's ok to import multiple times (i do it in java tests) and also this code won't be called if test fails

Reply:
This is a cleanup step, we did it in every test that create a new repository. I think we can remove this line once we refactor to share the same repository across tests.